lib/arrow.cpp

```diff
@@ -2036,9 +2037,8 @@ namespace grnarrow {
     void
     add_metadata(const char *key, const char *value)
```

Comment:
Would it be better to pass a hash of metadata?

Reply:
We can keep this signature. If we add the signature, caller needs to build a hash table something like `grn_hash`. It's not convenient and cheep.